spec/models/post_spec.rb

```diff
@@ -0,0 +1,197 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe Post, type: :model do
+  let!(:post) {FactoryBot.build :post}
+  let!(:post_fail) {FactoryBot.build :post, name: ""}
+
+  describe "Validations" do
+    context "when valid object" do
+      it "required all field" do
```

Comment:
describle, context, it là gì bạn nhỉ

Reply:
it: dùng để mô tả test case
describe: dùng để mô tả một nhóm các test case theo class, method (action trong controller)
context: để nhóm theo điều kiện của nhóm testcacse